app/invocation/cache_requests_card.tsx

```diff
@@ -657,6 +700,24 @@ export default class CacheRequestsCardComponent extends React.Component<CacheReq
                       )}
                     </>
                   )}
+                  {group.results[0]?.targetId && group.results[0]?.targetId.startsWith("//") && (
+                    <>
+                      <Tooltip
+                        className="row"
+                        pin={pinBottomMiddleToMouse}
+                        renderContent={() => (
+                          <div className="cache-result-hovercard">
+                            <div>Why did this target build?</div>
+                          </div>
+                        )}>
+                        <HelpCircle
+                          onClick={this.executeRemoteBazelQuery.bind(this, group.results[0]?.targetId!)}
+                          className="download-button icon"
+                          role="button"
+                        />
+                      </Tooltip>
+                    </>
+                  )}
```

Comment:
should we add this to the "Target" page instead (or in addition)?

Reply:
Done. I like having it on the cache table as well, because that seems more commonly used by customers